templates/plate-chain/plate_chain/chain.py

```diff
@@ -0,0 +1,91 @@
+import json
+import base64
+
+from langchain.chat_models import ChatOpenAI
+from langchain.prompts import ChatPromptTemplate, SystemMessagePromptTemplate
+from langchain.schema.output_parser import StrOutputParser
+from langchain.pydantic_v1 import BaseModel, Field
+from langserve import CustomUserType
+
+from .prompts import (
+    AI_REPONSE_DICT,
+    FULL_PROMPT,
+    USER_EXAMPLE_DICT,
+    create_prompt,
+)
+from .utils import parse_llm_output
+
+llm = ChatOpenAI(temperature=0, model="gpt-4")
+
+prompt = ChatPromptTemplate.from_messages(
+    [
+        SystemMessagePromptTemplate.from_template(FULL_PROMPT),
+        ("human", "{user_example}"),
+        ("ai", "{ai_response}"),
+        ("human", "{input}"),
+    ],
+)
+
+
+# ATTENTION: Inherit from CustomUserType instead of BaseModel otherwise
+# the server will decode it into a dict instead of a pydantic model.
+class FileProcessingRequest(CustomUserType):
+    """Request including a base64 encoded file."""
+
+    # The extra field is used to specify a widget for the playground UI.
+    file: str = Field(..., extra={"widget": {"type": "base64file"}})
+    num_plates: int = None
+    num_rows: int = 8
+    num_cols: int = 12
+
+
+def _load_file(request: FileProcessingRequest):
+    return str(base64.b64decode(request.file.encode("utf-8")))
```

Comment:
Need to decode('utf-8') rather than use `str` -- different results

```suggestion
    return base64.b64decode(request.file.encode("utf-8")).decode('utf-8')
```